packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -174,8 +174,18 @@ export default abstract class SqlIntegration
       (mo) => mo.id === metric.id
     );
     if (metricOverride) {
-      metric.conversionDelayHours = metricOverride.conversionDelayHours;
-      metric.conversionWindowHours = metricOverride.conversionWindowHours;
+      if ("conversionDelayHours" in metricOverride) {
+        metric.conversionDelayHours = metricOverride.conversionDelayHours;
+      }
+      if ("conversionWindowHours" in metricOverride) {
+        metric.conversionWindowHours = metricOverride.conversionWindowHours;
+      }
+      if ("winRisk" in metricOverride) {
```

Comment:
technically the "winRisk" and "loseRisk" overrides are not needed to generate these queries, but they're included for completeness.